cs-cz/javascript.html.markdown

```diff
@@ -9,32 +9,27 @@ lang: cs-cz
 filename: javascript-cz.js
 ---
 
-JavaScript byl vytvořen Brendan Eichem v roce 1995 pro Netscape. Byl původně
-zamýšlen jako jednoduchý skriptovací jazyk pro webové stránky, jako doplněk Javy,
-která byla zamýšlena pro více komplexní webové aplikace, ale jeho úzké propojení
-s webovými stránkami a vestavěná podpora v prohlížečích způsobila, že se stala
-více běžná ve webovém frontendu než Java.
+JavaScript byl vytvořen Brendanem Eichem v roce 1995 pro Netscape. Původně byl
+zamýšlen jako jednoduchý skriptovací jazyk pro webové stránky, jako doplněk
+Javy, která byla zamýšlena pro komplexnější webové aplikace. Úzké propojení
+JavaScriptu s webovými stránkami a vestavěná podpora v prohlížečích způsobila,
+že se stal ve webovém frontendu běžnějším než Java.
 
-
-JavaScript není omezen pouze na webové prohlížeče, např. projekt Node.js,
+JavaScript není omezen pouze na webové prohlížeče. Např. projekt Node.js,
 který zprostředkovává samostatně běžící prostředí V8 JavaScriptového enginu z
-Google Chrome se stává více a více oblíbený pro serverovou část webových aplikací.
-
-Zpětná vazba je velmi ceněná. Autora článku můžete kontaktovat (anglicky) na
-[@adambrenecki](https://twitter.com/adambrenecki), nebo
-[adam@brenecki.id.au](mailto:adam@brenecki.id.au), nebo mě, jakožto překladatele,
-na [martinek@ludis.me](mailto:martinek@ludis.me).
+Google Chrome se stává stále oblíbenější i pro serverovou část webových 
+aplikací.
 
 ```js
```

Comment:
Are you sure that block needs to be deleted as a whole in Czech?

Reply:
The block is not in source text. Also this duplicates text as author is also mentioned in footer. ludis.me domain does not exist for more than a year. I think we can remove the mail address. Author still remains mentioned in translators section.